test/test_base.py

```diff
@@ -119,6 +119,11 @@ def start_server(self):
     def stop_server(self):
         self.cleanup()
 
+    def start_network(self, file=None):
+        graph_file_path = (Path(__file__).parent / "data" / file) if file else DEFAULT_TEST_GRAPH
```

Comment:
why not `file=DEFAULT_TEST_GRAPH`  and `DEFAULT_TEST_GRAPH=v25_x_12.graphml` ?

Reply:
Done